### PR TITLE
Fix index creation

### PR DESCRIPTION
**Pull request by silvanheller**
This PR fixes indices not being created (#282), and has also led to two documentation / testing requests to the cottontail API (https://github.com/vitrivr/cottontaildb-proto/issues/9) and cottontail itself (https://github.com/vitrivr/cottontaildb/issues/105)

### CODE REVIEW

**Review by Spiess — APPROVED** [review]

**Comment by lucaro** [comment]
Given the recent changes on the API, this will become incompatible soon. So it might make sense to wait until the next version of cottontail to merge this.

**Comment by Spiess** [comment]
I suggest merging this now, so it is already fixed for the current version of Cottontail.
Compatibility can be restored in a separate, proper upgrade branch that bumps the minor version.

**Comment by silvanheller** [comment]
This needs merging now as the current behavior on master is broken (Indices are not created)

**Pull request merged**
